analyzer/pkg/dpkg/dpkg.go

```diff
@@ -144,5 +158,5 @@ func (a debianPkgAnalyzer) parseDpkgPkg(scanner *bufio.Scanner) (pkg *analyzer.P
 }
 
 func (a debianPkgAnalyzer) RequiredFiles() []string {
-	return []string{"var/lib/dpkg/status"}
+	return []string{"var/lib/dpkg/status", "var/lib/dpkg/status.d/"}
```

Comment:
```suggestion
	return append(a.requiredFiles(), a.requiredDirs()...)

func (a debianPkgAnalyzer) requiredFiles() []string {
	return []string{"var/lib/dpkg/status"}
}	
func (a debianPkgAnalyzer) requiredDirs() []string {
	return []string{"var/lib/dpkg/status.d/"}
}
```

If you care about a performance, we can define these variables in advance.